### DISCUSSION TITLE
Restarting simulation with different mesh

### DISCUSSION THREAD

**Discussion by fernandadialzira** [esm-tools/esm_tools#1205] [General]
Hi! 

I need to perform a simulation of the Late Pliocene using the modified `core2` mesh, therefore, with a different `nx` number. Apparently, I cannot branchoff my simulation from my PICTRL because the different meshes. Is this really the reason? Then my only option is to perform a long spinup of my Late Pliocene state?

Runscript: `/work/ab0246/a270179/runtime/run_configs/v32-midpli1.yaml`
Error when performing the simulation: `/work/ab0246/a270179/runtime/awicm3-v3.2/mp_003//run_23500101-23501231/log/mp_003_awicm3_compute_11233167.log`

```
Survived cpl_oasis3mct_define
   0:  FESOM ---->     cpl_oasis3mct_define_unstr nsend, nrecv:           7          13
 639:  fesom oasis_enddef: COMPLETED
   0:  ^[[1;33m --> read restarts from netcdf file: ocean^[[0m
 128:  reading restart PARALLEL for ssh at /work/ab0246/a270179/runtime//awicm3-v3.2//mp_003/run_23500101-23501231/work/fesom.2349.oce.restart/ssh.nc
 128:  error in line          565 io_netcdf_file_module.F90
 128: 1
 768: Computing intersections ...
srun: error: l40158: task 128: Exited with exit code 1
srun: Terminating StepId=11233167.0
   0: slurmstepd: error: *** STEP 11233167.0 ON l40157 CANCELLED AT 2024-07-19T14:13:39 ***
 768: Computing grads ...
 128: slurmstepd: error:  mpi/pmix_v3: _errhandler: l40158 [1]: pmixp_client_v2.c:212: Error handler invoked: status = -25, source = [slurm.pmix.11233167.0:128]
srun: Job step aborted: Waiting up to 62 seconds for job step to finish.
```
And this basically happens because `ssh.nc` in the `pictrl` is:

```
dimensions:
	time = UNLIMITED ; // (1 currently)
	node = 126858 ;
variables:
	double time(time) ;
	int iter(time) ;
	double ssh(time, node) ;
		ssh:units = "m" ;
		ssh:long_name = "sea surface elevation" ;
```

while for `late pliocene` is:
```
dimensions:
	time = UNLIMITED ; // (1 currently)
	node = 134452 ;
variables:
	double time(time) ;
	int iter(time) ;
	double ssh(time, node) ;
		ssh:units = "m" ;
		ssh:long_name = "sea surface elevation" ;
```

right?

Thank you in advance!

**Comment by JanStreffing** [comment]
Hey Fernanda, are actually you trying to restart from core2 with the pliocene mesh? Or is this just an esm_tools nx issue?

**Reply by fernandadialzira** [reply]
I am actually trying to restart from core2 with the pliocene mesh, but once it did not work, I decided to ask if it is even possible, or if I just need to do a long pliocene spinup.

**Comment by pgierz** [comment]
> Apparently, I cannot branchoff my simulation from my PICTRL because the different meshes.

TLDR: Yup! That's numerically unsupported by the model. If your mesh geometry between the two runs is different, there is no clear way to get the information to the right place.

- - - -

The longer story: available upon request, but after the weekend 😉

**Reply by fernandadialzira** [reply]
Achso!! Thank you :) now I know, so I will just proceed with the long Pliocene spinup
